src/domain/auth/state/authSelectors.tsx

```diff
@@ -0,0 +1,14 @@
+import { createSelector } from 'reselect';
+
+import { StoreState } from '../../app/types/stateTypes';
+
+export const idTokenSelector = (state: StoreState) =>
+  state.authentication.user && !!state.authentication.user.id_token;
```

Comment:
You probably want to remove `!!` from `!!state.authentication.user.id_token;` here

Reply:
true, i tested and forgot to remove, i merge too fast, thank for pointing out!

Reply:
I'll try to be faster next time!